docs/UsersGuide/source/ConfigWorkflow.rst

```diff
@@ -136,109 +181,427 @@ These variables apply only when using NCO mode (i.e. when ``RUN_ENVIR`` is set t
+   
+   **Current supported settings for this parameter are:** 
+
+   | "FV3_GFS_v16" 
+   | "FV3_RRFS_v1beta" 
+   | "FV3_HRRR"
+   | "FV3_WoFS"
+
+   **Other valid values include:**
+
+   | "FV3_GFS_2017_gfdlmp"
+   | "FV3_GFS_2017_gfdlmp_regional"
+   | "FV3_GFS_v15p2"
+   | "FV3_GFS_v15_thompson_mynn_lam3km"
+   | "FV3_RRFS_v1alpha"
+
+..
+   COMMENT: "FV3_WoFS" technically has not been merged yet... and is called NSSL? What should I put for now? Current Default is "FV3_GFS_v15p2" - need to make sure we change that. 
+
+Stochastic Physics Parameters
+================================
+
+For the most updated and detailed documentation of these parameters, see the `UFS Stochastic Physics Documentation <https://stochastic-physics.readthedocs.io/en/latest/namelist_options.html>`__.
+
+
+``NEW_LSCALE``: (Default: "TRUE") 
+   Use correct formula for converting a spatial legnthscale into spectral space. 
+
+Specific Humidity (SHUM) Perturbation Parameters
+---------------------------------------------------
+
+``DO_SHUM``: (Default: "FALSE")
+   Flag to turn Specific Humidity (SHUM) perturbations on or off. SHUM perturbations multiply the low-level specific humidity by a small random number at each time-step. The SHUM scheme attempts to address missing physics phenomena (e.g., cold pools, gust fronts) most active in convective regions. 
+
+``ISEED_SHUM``: (Default: "2")
+   Seed for setting the SHUM random number sequence.
+
+``SHUM_MAG``: (Default: "0.006") 
+   Amplitudes of random patterns. Corresponds to the variable ``shum`` in ``input.nml``.
+
+``SHUM_LSCALE``: (Default: "150000")
+   Decorrelation spatial scale in meters.
+
+``SHUM_TSCALE``: (Default: "21600")
+   Decorrelation timescale in seconds. Corresponds to the variable ``shum_tau`` in ``input.nml``.
+
+``SHUM_INT``: (Default: "3600")
+   Interval in seconds to update random pattern (optional). Perturbations still get applied at every time-step. Corresponds to the variable ``shumint`` in ``input.nml``.
+
+.. _SPPT:
+
+Stochastically Perturbed Physics Tendencies (SPPT) Parameters
+-----------------------------------------------------------------
+
+SPPT perturbs full physics tendencies *after* the call to the physics suite, unlike :ref:`SPP <SPP>` (below), which perturbs specific tuning parameters within a physics scheme. 
+
+``DO_SPPT``: (Default: "FALSE")
+   Flag to turn Stochastically Perturbed Physics Tendencies (SPPT) on or off. SPPT multiplies the physics tendencies by a random number between 0 and 2 before updating the model state. This addresses error in the physics parameterizations (either missing physics or unresolved subgrid processes). It is most active in the boundary layer and convective regions. 
+
+..
+   COMMENT: Does it need to be O? can it be X? or some other variable/letter that doesn't look like the number zero? 
```

Comment:
Does what need to be zero, the seed?  If so, the seed needs to be some kind of number.

Reply:
Oops! Forgot to remove that comment. Resolved it earlier with Phil's help. Sorry about that.